main.yml

```diff
@@ -581,16 +581,22 @@ openshift:
     paths:
       - /openshift
     sub_apps:
+      - id: overview
```

Comment:
The id should be `''` otherwise it would break the openshift flow.

Reply:
Oh, the overview just moved up! Let's see what will do this reoder in the URL and navigation